Documentation/Releases/solr-release-7-0.rst

```diff
@@ -17,29 +17,335 @@
+* Core Optimization: This module can be used to maintain the synonyms and stopwords in the Apache Solr server.
+* Index Queue: Gives an overview on indexed records and can be used to requeue records for indexing.
+* Index Administration: This module can be used for administrator tasks on your solr system (clear index, index queue or reload a core)
+
+* https://github.com/TYPO3-Solr/ext-solr/pull/1300
+
+
+Add excludeValues for Facets
+----------------------------
+
+If you want to exclude an optionValue from the facets when they get retrieved, you can configure this now:
+
+.. code-block:: typoscript
+
+   plugin.tx_solr.search.faceting.facets.colors_stringM.excludeValues = red
+
+
+The example below will exclude the option "red" from the results when it is in the response.
+
+* https://github.com/TYPO3-Solr/ext-solr/pull/1364
+
+Allow to configure custom entry Template
+----------------------------------------
+
+By convention the actionName is used in extbase for the entry template. You can now configure this template to use another entry template.
+This is useful, when you setup multiple instances with own entry templates.
+
+* https://github.com/TYPO3-Solr/ext-solr/pull/1325
+
+Refactoring of Query API
+------------------------
+
+The Query class is one of the biggest classes in EXT:solr and grown over time. This class has now been splitted into several classes.
+Along with that a concept of "ParameterBuilder" has been introduced. A ParameterBuilder is responsible to build a parameter part of the query.
+E.g. the Grouping ParameterBuilder is responsible to build all parameters of the solr query for the grouping.
+
+* https://github.com/TYPO3-Solr/ext-solr/pull/1385
+
+Move FilterEncoder and FacetBuilder to Facet Package
+----------------------------------------------------
+
+In Solrfluid there was one folder for each facet, that contains the facet class and a parser that parsers the solr response into the facet object.
+The opposite part(parse the url, build the solr query) was previously done in EXT:solr, with a FilterEncoder that was registered in the FacetRendererFactory.
+
+Now because solrfluid and solr have been merged, this logic can also be streamlined. Every facet is now structured in a FacetPackage.
+
+A FacetPackage describes:
+
+* Which parser should be used to parse the solr response
+* Which url decoder should be used to parse the EXT:solr query data
+* Which query builder should be used to build the faceting query part
+
+You can also implement custom facet types be registering an on FacetPackage with the FacetRegistry.
+
+**Migration**:
+
+When you have implemented an own FacetParser for solrfluid, you should add a FacetPackage, that references a UrlDecoder and QueryBuilder.
+If you have used a custom FacetParser without registring a custom facet type in EXT:solr (ApacheSolrForTypo3\Solr\Facet\FacetRendererFactory::registerFacetType) you can just reference DefaultUrlDecorder and DefaultFacerQueryBuilder in you FacetPackage.
```

Comment:
"... reference DefaultUrlDecorder and DefaultFacerQueryBuilder in **your** FacetPackage"